File: 2022 test/day3/P2.java
import java.io.*;  

class Main {
    public static void main(String[] argv){
        try{
            File file = new File("input.txt");
            FileReader fr = new FileReader(file);   //reads the file  
            BufferedReader br = new BufferedReader(fr);  //creates a buffering character input stream
            int sum = 0;

            while (true){
                String elf_1 = br.readLine();
                if (elf_1==null){
                    break;
                }
                String elf_2 = br.readLine();
                String elf_3 = br.readLine();
                String badge;

                int len_elf_1 = elf_1.length();
                int len_elf_2 = elf_2.length();
                int len_elf_3 = elf_3.length();

                System.out.println(len_elf_1);
                System.out.println(len_elf_2);
                System.out.println(len_elf_3);                
            }
            fr.close();
            br.close();
        }catch(IOException e){  
            e.printStackTrace();  
            }  
    }
}

File: 2022 test/day3/P2.py
file = open("input.txt", 'r')
sum = 0
while (True):
    elf_1 = file.readline()
    if (elf_1 == ""):
        break
    elf_2 = file.readline()
    elf_3 = file.readline()

    badge = ""

    len_elf_1 = len(elf_1)-1
    len_elf_2 = len(elf_2)-1
    len_elf_3 = len(elf_3)-1

    priority = -1

    for i in range(len_elf_1):
        for j in range(len_elf_2):
            for k in range(len_elf_3):
                

    print(badge, priority)
    sum+=priority
print(sum)